src/components/cart.js

```diff
@@ -317,7 +317,9 @@ export default class Cart extends Component {
    * Remove all lineItems in the cart
    */
   empty() {
-    return this.model.clearLineItems().then(() => {
+    const lineItemIds = this.model.lineItems.map((item) => item.id);
+
+    return this.props.client.removeLineItems(this.model.id, lineItemIds).then(() => {
```

Comment:
this returns a new checkout instance, which should maybe replace `model`?

Reply:
so set `this.model` to this before returning it itself?